--- jailkit-2.13/ini/jk_init.ini~	2010-09-11 15:58:41.000000000 +0200
+++ jailkit-2.13/ini/jk_init.ini	2010-11-11 15:44:06.560790682 +0100
@@ -48,7 +48,7 @@
 
 [sftp]
 comment = ssh secure ftp
-paths = /usr/lib/sftp-server, /usr/libexec/openssh/sftp-server, /usr/lib/misc/sftp-server, /usr/libexec/sftp-server
+paths = /usr/lib/sftp-server, /usr/libexec/openssh/sftp-server, /usr/lib/misc/sftp-server, /usr/libexec/sftp-server, /usr/lib/openssh/sftp-server
 includesections = netbasics, uidbasics
 devices = /dev/urandom, /dev/null
 # on solaris 
